core/src/test/scala/integration/kafka/api/ProducerBounceTest.scala

```diff
@@ -61,9 +63,9 @@ class ProducerBounceTest extends KafkaServerTestHarness {
   override def setUp() {
     super.setUp()
 
-    producer1 = TestUtils.createNewProducer(brokerList, acks = 0, bufferSize = producerBufferSize)
-    producer2 = TestUtils.createNewProducer(brokerList, acks = 1, bufferSize = producerBufferSize)
-    producer3 = TestUtils.createNewProducer(brokerList, acks = -1, bufferSize = producerBufferSize)
+    //producer1 = TestUtils.createNewProducer(brokerList, acks = 0, bufferSize = producerBufferSize)
```

Comment:
I think I should work multiple producers into the test, since there would be bugs that such a test would catch, as you previous comments suggest :)